packages/graphql/tests/tck/connections/connect-or-create/connect-or-create.test.ts

```diff
@@ -178,7 +178,7 @@ describe("Create or Connect", () => {
             typeDefs = gql`
                 type Movie {
                     id: ID! @id @unique
-                    createdAt: DateTime! @timestamp(operations: [CREATE])
+                    createdAt: DateTime @timestamp(operations: [CREATE])
```

Comment:
You should be able to leave these as non-nullable!

```suggestion
                    createdAt: DateTime! @timestamp(operations: [CREATE])
```